rasa_core/policies/keras_policy.py

```diff
@@ -70,7 +70,7 @@ def _build_model(self, num_features, num_actions, max_history_len):
         # Build Model
         model = Sequential()
         model.add(Masking(-1, batch_input_shape=batch_shape))
-        model.add(LSTM(n_hidden, batch_input_shape=batch_shape))
+        model.add(LSTM(n_hidden, batch_input_shape=batch_shape, dropout=0.2))
```

Comment:
please double check whether this means "keep 20%" or "drop 20%" - varies between DL frameworks

Reply:
that is the fraction to drop